src/terrain/tile.cpp

```diff
@@ -157,7 +157,7 @@ Tile::get_mat_color_id() const {
 }
 
 // return x, y, z positions as array
-std::array<uint16_t, 3>
+std::array<Dim, 3>
```

Comment:
glm vector

Reply:
I think I said this, but I can't find where. I don't want to use glm vec3 unless I can automatically set the size of each index to be the same as the size Dim.

Reply:
You did not say that. Just put your typedefs next to each other, and add a static assert. That way its obvious when you need to change them.